src/style.css

```
@@ -1 +1,81 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background-color: #fff;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.logo img {
+  margin: 0;
+  padding: 0;
+  width: 40px;
+  height: 40px;
+}
+
+ul.list {
+  margin: 0;
+  padding: 0;
+}
```

1) you can apply reset styles to your tags
2) move these reset styles to the top of your file
```suggestion
ul {
  margin: 0;
  padding: 0;
  list-style: none;
}
```